unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -69,6 +75,56 @@ class TimeTest {
     assertTrue(now_time_t > 1502752688)
   }
 
+  @Test def strftimeDoesNotReadMemoryOutsideStructTm(): Unit = {
+    Zone { implicit z =>
+      // Review logic of this test thoroughly if size of "tm" changes.
+      // This test may no longer be needed or need updating.
+      assumeTrue("Assumes 36 byte Scala Native struct tm",
+                 (sizeof[tm] == 36.toULong))
```

Comment:
An assumption causes the test to be silently ignored if it evaluates to false. That means that this test will simply stop executing if the size changes, which means we won't notice bugs.

This should be a hard `assert` instead.